src/main/components/uml-element/assessable/assessable.tsx

```diff
@@ -154,7 +154,10 @@ export const assessable = (
         const { id: elementId, assessment } = this.props;
         const score = instruction.credits;
         const feedback = instruction.feedback;
-        const dropInfo = { instructionId: instruction.id };
+        const dropInfo = {
+          instructionId: instruction.id,
+          instructionDetail: instruction.instructionDescription,
+        };
```

Comment:
Instructions should not go outside this function.

Reply:
so should I pass the tooltip msg from Artemis like `dropInfo.tooltip`, then I won't use any translation for grading instructions... will look into the initializing the editor from Artemis side

Reply:
You can give the tooltip according to the language selected in Artemis or define them inside the dropEvent function with a TODO to remove later but outside of this function there shouldn't be anything instruction related.